docs/md/iofogctl_configure.md

```diff
@@ -17,7 +17,6 @@ iofogctl configure resource NAME [flags]
 ```
 iofogctl configure default-namespace NAME
 iofogctl configure controller NAME --host HOST --user USER --key KEYFILE --port PORTNUM
-iofogctl configure controller NAME --kube KUBECONFIG
```

Comment:
Our code was never assigning this value and our tests were never checking it. We can re-add this when we add functionality + test